core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -369,7 +364,8 @@ static struct tee_mmap_region *init_xlation_table(struct tee_mmap_region *mm,
 						 level);
 				debug_print("%*s%010" PRIx64 " %8x %s-%s-%s-%s",
 					level * 2, "", base_va, level_size,
-					attr & TEE_MATTR_CACHE_DEFAULT ?
+					attr & (TEE_MATTR_CACHE_CACHED <<
```

Comment:
This is analogous with the debug prints in `init_xlation_table()`, and also the strings are shorter so the logs are easier to read.